gensim/models/wrappers/fasttext.py

```diff
@@ -123,11 +123,8 @@ def __contains__(self, word):
         if word in self.vocab:
             return True
         else:
-            word_ngrams = set(FastText.compute_ngrams(word, self.min_n, self.max_n))
-            if len(word_ngrams & set(self.ngrams.keys())):
-                return True
-            else:
-                return False
+            word_ngrams = FastText.compute_ngrams(word, self.min_n, self.max_n)
+            return any(ng in self.ngrams for ng in word_ngrams)
```

Comment:
Agreed. Since `dict.__contains__` is fast (relative to allocating and filling in a new `set` object), and the expected duplication rate low (I think), the current approach is probably for the best.

IMO the `set` optimization should come from `compute_ngrams` itself, to avoid creating the intermediate list. An iterator over unique ngrams would be the best -- no explicit list or set needed (since we short-circuit anyway). CC @jayantj